spec/features/notifications_spec.rb

```diff
@@ -3,26 +3,34 @@
 RSpec.describe "Notifications" do
   describe " visit notifications" do
     before (:each) { sign_in_as('Editor') }
-    let!(:notification)  { create(:notification, subject: "Howdy from Hopedale") }
+    let(:department) { create(:department) }
+    let!(:notification)  { create(:notification,
+                                  subject: "Howdy from Hopedale",
+                                  departments: [department]) }
     context "basic CRUD" do
       it "should display a single notification" do
         visit notification_path(notification)
         expect(page).to have_content("Howdy")
       end
     end
     context "from an event" do
-      let!(:event)  { create(:event) }
+      let!(:valid_department)  { create(:department, manage_people: true, name: 'Valid Dept') }
+      let!(:invalid_department)  { create(:department, manage_people: false, name: 'Invalid Dept') }
+      let!(:event)  { create(:event, departments: [valid_department]) }
```

Comment:
We used `Event.new` previously. Do we have to create this in the db or can we get away with using `build` instead of `create` for the same effect?

Reply:
I tried it, no joy. Since this is a feature spec, and we're trying to visit the show page, the event must be persisted.